scripts/zephyr_module.py

```diff
@@ -257,27 +257,32 @@ def process_blobs(module, meta):
     return blobs
 
 
-def kconfig_snippet(meta, path, kconfig_file=None):
+def kconfig_snippet(meta, path, kconfig_file=None, blobs=False):
     name = meta['name']
     name_sanitized = meta['name-sanitized']
 
-    snippet = (f'menu "{name} ({path.as_posix()})"',
+    snippet = [f'menu "{name} ({path.as_posix()})"',
                f'osource "{kconfig_file.resolve().as_posix()}"' if kconfig_file
                else f'osource "$(ZEPHYR_{name_sanitized.upper()}_KCONFIG)"',
                f'config ZEPHYR_{name_sanitized.upper()}_MODULE',
                '	bool',
                '	default y',
-               'endmenu\n')
+               'endmenu\n']
+
+    if blobs:
+        snippet.insert(-1, '	select TAINT_BLOBS')
     return '\n'.join(snippet)
 
 
 def process_kconfig(module, meta):
+    blobs = process_blobs(module, meta)
+    taint_blobs = len(tuple(filter(lambda b: b['status'] != 'D', blobs))) != 0
```

Comment:
A little bit of magic string use here is not so readable. It would be better to use the named constant for 'D', or even use BLOB_PRESENT instead of 'not BLOB_NOT_PRESENT'. Maybe something like this?

```py
taint_blobs = any([lambda b: b['status'] == BLOB_PRESENT for b in blobs])
```

Reply:
True, I will let @tejlmand follow-this one up with the rest. Thanks for the suggestion!

Reply:
Yep, makes sense. Still would rather have this use a named constant instead of a magic value.